test/hotspot/jtreg/vmTestbase/nsk/jvmti/ResourceExhausted/resexhausted001.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2007, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2007, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2007, 2025, Oracle and/or its affiliates. All rights reserved.
```

Reply:
subconsciously still in vacation, I guess. 😄